### ISSUE TITLE
Create documentation for player class demo

### ISSUE THREAD

**Issue by vanolson** [uchicago-cs/chiventure#1620]
Most of the documentation for the player class demo is currently in the comments. A lot of it still holds true, since we're demoing similar items as last year and can therefore reuse a number of the functions from last year. We will create a wiki page documenting the demo, including details both on how to use it and how it works. We will need to document both the multiclass demo and prefab class demo, with potentially additions regarding multiclass skills as we build on the demo. Jake, Katie, and I will likely add to and edit this as we work on the demo.

**Comment by vanolson** [comment]
I've written a preliminary version of the page, found [here](https://github.com/uchicago-cs/chiventure/wiki/Player-Class-~-Class-Demo-Documentation). We will continue to update it as we develop the demo.

**Comment by wellingtj** [comment]
We have finalized the page. It is found in the same location as where Van linked earlier on the Wiki

**Comment by vanolson** [comment]
Closing comment: I submitted the edits, but Katie and Jake were heavily involved both in figuring out the demo's workings from last year and evolving it to this year, so they were instrumental to the development of these processes. We documented both the prefab class portion of the demo and the multiclass portion, in addition to providing instructions for getting the demo running and user processes for going through the demo. We also provided a short section on how the demo is structured, both regarding its files and function structure.
